packages/components/switch/src/Switch.variants.tsx

```diff
@@ -0,0 +1,65 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+const defaultVariants = {
+  intent: 'primary',
+  size: 'small',
+} as const
+
+export const styles = cva(
+  [
+    'group relative inline-flex flex-shrink-0',
+    'cursor-pointer',
+    'rounded-full border-md border-transparent',
+    'transition-colors duration-200 ease-in-out',
+    'radix-disabled:opacity-50 radix-disabled:cursor-not-allowed',
+    'focus:outline-none focus-visible:ring focus-visible:ring-outline-high',
+  ],
+  {
+    variants: {
+      size: {
+        small: ['h-[1rem]', 'w-[2.25rem]'],
```

Comment:
se should avoid using arbitrary values, as it may lead to UI inconsistencies if a different base font size is used at the call site